tests/integration/ipsec_test.py

```diff
@@ -754,6 +753,13 @@ def test_ipsec_ipv4_libreswan_psk_auth_with_ipsec_iface(
     assert retry_till_true_or_timeout(
         RETRY_COUNT, _check_ipsec_ip, HOSTB_VPN_SUBNET_PREFIX, "ipsec9"
     )
+    yield
+
+
+def test_ipsec_ipv4_libreswan_psk_auth_with_ipsec_iface(
+    ipsec_psk_with_ipsec_iface,
+):
+    pass
```

Comment:
Shouldn't we assert that everything is fine here? Otherwise we are only relying on verification which could be bugged.

Reply:
My concern is only calling the fixture in a test, why don't we just drop this test, because `test_ipsec_ipv4_libreswan_change_ipsec_iface` also uses the fixture - `ipsec_psk_with_ipsec_iface`.

Reply:
The fixture `ipsec_psk_with_ipsec_iface` already contains the assertion.

I have remove this test case as duplication.